src/main/java/core/basesyntax/AreaCalculate.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculate {
```

Comment:
```suggestion
public interface AreaCalculator {
```
interface and class names should better be nouns